spec/capitalizedecorator_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../capitalizedecorator'
+
+describe CapitalizeDecorator do
+  before :each do
+    @capitalize_decorator = CapitalizeDecorator.new('Ichsan')
+  end
+
+  it 'should return an instance of CapitalizeDecorator' do
+    expect(@capitalize_decorator).to be_an_instance_of CapitalizeDecorator
+  end
```

Comment:
- [ ] Here you are supposed to test if the Capitalize method works, for example, if you have `ichsan` as name it is supposed to correct the name by using the `correct_name` method and should be eq `Ichsan` 🙃